bevy_rapier2d/examples/interpolation2.rs

```diff
@@ -0,0 +1,116 @@
+use std::{fs::File, io::Write};
+
+use bevy::{color::palettes, prelude::*};
+use bevy_mod_debugdump::{schedule_graph, schedule_graph_dot};
+use bevy_rapier2d::prelude::*;
+use bevy_transform_interpolation::prelude::{
+    RotationInterpolation, TransformInterpolationPlugin, TranslationInterpolation,
+};
+
+fn main() {
+    let mut app = App::new();
+    app.insert_resource(ClearColor(Color::srgb(
+        0xF9 as f32 / 255.0,
+        0xF9 as f32 / 255.0,
+        0xFF as f32 / 255.0,
+    )))
+    .insert_resource(TimestepMode::Variable {
+        max_dt: 100f32,
+        time_scale: 1f32,
+        substeps: 10,
+    })
+    .insert_resource(Time::<Fixed>::from_hz(5.0))
+    .add_plugins((
+        DefaultPlugins,
+        TransformInterpolationPlugin::default(),
+        RapierPhysicsPlugin::<NoUserData>::pixels_per_meter(100.0).in_fixed_schedule(),
```

Comment:
`bevy_transform_interpolation` wouldn't work if we used `Update` ; but it makes no sense to interpolate when using `Update` schedule.